### DISCUSSION TITLE
Code completion and ctrl-clicking not working

### DISCUSSION THREAD

**Discussion by markdemich** [BowlerHatLLC/vscode-as3mxml#681] [Q&A]
I have the extension installed, but my code completion and ctrl-clicking is not working.  Where can I look for clues to fix this?

I went to the OUTPUT window under ActionScript & MXML Language Server and I see this.  Maybe it's a clue?
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.dom4j.io.SAXContentHandler (file:/C:/Users/demi/.vscode/extensions/bowlerhatllc.vscode-as3mxml-1.15.0/bin/dom4j-1.6.1.jar) to method com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser$LocatorProxy.getEncoding()
WARNING: Please consider reporting this to the maintainers of org.dom4j.io.SAXContentHandler
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Apr 09, 2023 8:20:40 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 34
Apr 09, 2023 8:21:10 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 37
Apr 09, 2023 8:31:55 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 137
```

I also noticed I no longer have the "Restart As3mxl server" option in the Run Task menu.

**Comment by joshtynjala** [comment]
>  went to the OUTPUT window under ActionScript & MXML Language Server and I see this. Maybe it's a clue?

This is normal output. Nothing to worry about there. It seems like the server has started correctly.

> I also noticed I no longer have the "Restart As3mxl server" option in the Run Task menu.

It's not in Run Task. It's in the Command Palette (View->Command Palette or Ctrl+Shift+P).

**Reply by markdemich** [reply]
Any ideas on why the code completion and ctrl-clicking is not working?

**Reply by joshtynjala** [reply]
If something is wrong, you should see a notification in VSCode (the bell in bottom right corner), or you should see an error/info diagnostic in the Problems view.

It would be very difficult make a suggestion without reading some kind of error message first.
